frontend/test/validate-e2e-test-files.js

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env node
+const { execSync } = require("child_process");
+
+const path = require("path");
+const glob = require("glob");
+const chalk = require("chalk");
+
+const e2eFileExtension = ".cy.spec.js";
+const e2eHome = "frontend/test/metabase/scenarios/";
```

Comment:
nit: so we know this is a global variables
```suggestion
const E2E_FILE_EXTENSION = ".cy.spec.js";
const E2E_HOME = "frontend/test/metabase/scenarios/"; // or E2E_PATH
```